a/apache-httpd/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer":"Abhishek.Dwivedi6@ibm.com",
+    "package_name" : "httpd",
```

Comment:
here it should be "apache-httpd"

Reply:
Updated the package name to "apache-httpd"